client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -12,12 +12,20 @@ export default function Dashboard(): JSX.Element {
   const classes = useStyles();
 
   const { loggedInUser } = useAuth();
-  const { initSocket } = useSocket();
+  const { initSocket, socket } = useSocket();
 
   const history = useHistory();
 
   useEffect(() => {
     initSocket();
+  }, []);
+
+  useEffect(() => {
+    socket?.open();
```

Comment:
I don't think we need to keep opening and closing the socket on this page since we're using Context, we'll just keep it open the whole time

Reply:
Sure, I will modify the above code to keep the socket connection open throughout. Thanks!